### PR TITLE
Nova scheduler

### PR DESCRIPTION
**Pull request by cloudnull**
This PR adds sane limits in nova. Presently the nova scheduler is set to filter though has no set filters. This allows a user to spin up a potentially unlimited number of vm's which eventually causes the host to become overloaded. Additionally the unset scheduler seems to stack vm's on a single host instead of spreading them out on all compute nodes. While this PR provides for sane settings it also allows any scheduler tunable to be changed as needed.

### CODE REVIEW

File: rpc_deployment/inventory/group_vars/nova_all.yml [hunk 1]
```diff
@@ -67,6 +67,25 @@ nova_firewall_driver: nova.virt.firewall.NoopFirewallDriver
...
 
+# Nova Scheduler
+nova_cpu_allocation_ratio: 10.0
+nova_disk_allocation_ratio: 1.0
+nova_max_instances_per_host: 50
+nova_max_io_ops_per_host: 10
+nova_ram_allocation_ratio: 1.5
+nova_ram_weight_multiplier: 5.0
+nova_reserved_host_disk_mb: 1024
+nova_reserved_host_memory_mb: 512
```
Comment (galstrom21) [inline rpc_deployment/inventory/group_vars/nova_all.yml]: Might want to make this 1024

File: rpc_deployment/inventory/group_vars/nova_all.yml [hunk 2]
```diff
@@ -67,6 +67,25 @@ nova_firewall_driver: nova.virt.firewall.NoopFirewallDriver
 nova_compute_driver: libvirt.LibvirtDriver
 nova_max_age: 0
 
+# Nova Scheduler
+nova_cpu_allocation_ratio: 10.0
```
Comment (galstrom21) [inline rpc_deployment/inventory/group_vars/nova_all.yml]: supports default for this value is 2.0
https://github.com/rsoprivatecloud/openstack-chef-deploy/blob/master/environments/havana-neutron.json#L58

File: rpc_deployment/inventory/group_vars/nova_all.yml [hunk 3]
```diff
@@ -67,6 +67,25 @@ nova_firewall_driver: nova.virt.firewall.NoopFirewallDriver
...
+nova_disk_allocation_ratio: 1.0
```
Comment (galstrom21) [inline rpc_deployment/inventory/group_vars/nova_all.yml:72]: supports default for this value was 1.0
https://github.com/rsoprivatecloud/openstack-chef-deploy/blob/master/environments/havana-neutron.json#L59

File: rpc_deployment/inventory/group_vars/nova_all.yml [hunk 4]
```diff
@@ -67,6 +67,25 @@ nova_firewall_driver: nova.virt.firewall.NoopFirewallDriver
...
+nova_max_instances_per_host: 50
+nova_max_io_ops_per_host: 10
+nova_ram_allocation_ratio: 1.5
```
Comment (galstrom21) [inline rpc_deployment/inventory/group_vars/nova_all.yml:75]: supports default for this was 1.0
https://github.com/rsoprivatecloud/openstack-chef-deploy/blob/master/environments/havana-neutron.json#L57

**Comment by d34dh0r53** [comment]
LGTM

**Pull request merged**